test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -183,31 +183,6 @@ def test_deeplab(self):
         x = torch.randn(2, 3, 224, 224, requires_grad=True)
         self.run_test(model, (x,), rtol=1e-3, atol=1e-5)
 
-    def test_googlenet_quantization(self):
```

Comment:
curious why are these tests removed?

Reply:
They are not actually testing quantized version of models since quantization is not enabled. We are not planning to add ORT tests for quantized models for now.

Reply:
I asked Negin to remove these tests in this PR. 
The tests are for the quantization models that we actually don't support.
I initially added these tests, then noticed that we need to enable a boolean to run the quantized version. There are already tests for the non-quantized version, so these are duplicated.